libs/templates/src/lib/who-we-are/sections/mission-content/data.tsx

```diff
@@ -17,7 +17,7 @@ const Content = () => {
     <>
       {data.map((data, index) => {
         return (
-          <Text className="text-opacity-black-600" key={index}>
```

Comment:
please don't use `map index` as key in loop, please update it with some id or something else, thanks.

Reply:
Is there any reason not to use `map index` as key?